frontend/www/js/omegaup/components/contest/Edit.vue

```diff
@@ -363,13 +365,16 @@ export default class Edit extends Vue {
   alreadyArchived = this.details.archived;
 
   selectedTab(): string {
+    if (this.initialTab != '') {
+      return this.initialTab;
+    }
     if (!ui.isVirtual(this.details)) {
       return 'new_form';
     }
     if (this.originalContestAdmissionMode != 'private') {
       return 'contestants';
     }
-    return 'links';
+    return 'problems';
```

Comment:
Okay, I knew this value had a reason.

We have two kinds of virtual contests:

- Those in which the original contest's admission mode is public, allowing contestants to be added:
![image](https://github.com/omegaup/omegaup/assets/3230352/c6bdec9b-9b42-4b98-8c41-8eedceb77e61)

- Those in which the original contest's admission mode is private, restricting the addition of contestants:
![image](https://github.com/omegaup/omegaup/assets/3230352/4e5caa1c-8b1a-436f-8462-fc0c5d9e7bfc)

So, the return value should be `links`
```suggestion
    return 'links';
```

Reply:
No problem, @pabo99.